### PR TITLE
Various QoL for caretaker

### PR DESCRIPTION
**Pull request by hansl**
Just got a mistake (caught in time) so decided to do a long overdue refactoring for release script(s).

### CODE REVIEW

File: .monorepo.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "versions": {
+    "stable": "8.0.0-beta.5",
```
Comment (alexeagle) [inline .monorepo.json]: FYI on Angular we determine the version based on git tags; see https://github.com/angular/angular/blob/master/tools/bazel_stamp_vars.js

If we also had a scheme for making the 0.* versions based on the stable ones, then we'd only need one bit to describe the versioning.

That would help us move to bazel for build/publish someday

File: lib/packages.ts [hunk 2]
```diff
@@ -28,53 +26,19 @@ export interface PackageInfo {
   build: string;
   tar: string;
   private: boolean;
+  experimental: boolean;
```
Comment (alexeagle) [inline lib/packages.ts]: does this change mean we can update release.md to stop using --force?

Comment (hansl) [reply lib/packages.ts]: Yes, done.

File: .monorepo.json (hunk 1 above)
Comment (hansl) [reply .monorepo.json]: All done.

**Pull request merged**